app/scripts/pages/connections.js

```diff
@@ -42,28 +41,28 @@ var Connections = React.createClass({
           peer.ipfs = t.props.ipfs
           peer.location = { formatted: '' }
 
-          var location = t.state.locations[peer.ID]
-          if (!location) {
-            t.state.locations[peer.ID] = {}
-            t.props.ipfs.id(peer.ID, function (err, id) {
-              if (err) return console.error(err)
+          // var location = t.state.locations[peer.ID]
+          // if (!location) {
+          //   t.state.locations[peer.ID] = {}
+          //   t.props.ipfs.id(peer.ID, function (err, id) {
+          //     if (err) return console.error(err)
 
-              getLocation(t.props.ipfs, id.Addresses, function (err, res) {
-                if (err) return console.error(err)
-                // If we've unmounted, abort
-                if (!t.isMounted()) return
+          //     getLocation(t.props.ipfs, id.Addresses, function (err, res) {
+          //       if (err) return console.error(err)
+          //       // If we've unmounted, abort
+          //       if (!t.isMounted()) return
 
-                res = res || {}
-                peer.location = res
-                t.state.locations[peer.ID] = res
-                t.setState({
-                  peers: peers,
-                  locations: t.state.locations,
-                  nonce: t.state.nonce++
-                })
-              })
-            })
-          }
+          //       res = res || {}
+          //       peer.location = res
+          //       t.state.locations[peer.ID] = res
+          //       t.setState({
+          //         peers: peers,
+          //         locations: t.state.locations,
+          //         nonce: t.state.nonce++
+          //       })
+          //     })
+          //   })
+          // }
```

Comment:
Why is this commented out, and not deleted? Also; I can't see any peers on the globe; expected?